lib/pwa.js

```diff
@@ -198,8 +198,12 @@ exports.createOrUpdatePwa = function(pwa) {
       }
       // Delete modified PWA from cache
       const url = '/pwas/' + pwa.id;
-      cache.del(url);
-      cache.del(url + '?contentOnly=true');
+      cache.del(url).catch(_ => {
+        // URL not in cache.
```

Comment:
done, I removed both. I was trying to get rid of 2 promise unhandled warnings on the test, but this did not help.